bundles/pubsub/pubsub_utils/src/pubsub_skt_handler.c

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 /*
- * pubsub_tcp_handler.c
+ * pubsub_skt_handler.c
```

Comment:
Very nice to see that the socket concept can be shared between multiple PSA implementations.

Reply:
Can the psa_udp replace the psa_udp_mc. In other words can the psa_udp_mc implementation be removed?